tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -250,6 +250,28 @@ def test_ops_public_mpc(get_clients, nr_clients, op_str) -> None:
     assert np.allclose(result, expected_result, atol=10e-4)
 
 
+falcon = Protocol.registered_protocols["Falcon"]()
+
+
+@pytest.mark.parametrize("nr_clients", [3, 5, 7])
+@pytest.mark.parametrize("op_str", ["add", "sub"])
+def test_ops_public_tensor_rst(get_clients, nr_clients, op_str) -> None:
```

Comment:
Q: Does it make sense to also add a test where:
```
x = ShareTensor
y = ReplicatedSharingTensor
```
and vice-versa

Reply:
George You are meaning something like raising an error when share class are not equal right?